src/VSCodeExtension/package.json.v.template

```diff
@@ -74,7 +119,7 @@
                         "null"
                     ],
                     "description": "Path to the language server assembly for Q#.",
-                    "default": null
+                    "default": "C:/Users/t-jackmonas/work/qsharp-compiler/src/QsCompiler/LanguageServer/bin/Debug/net6.0/win10-x64/publish/Microsoft.Quantum.QsLanguageServer.exe"
```

Comment:
Don't include this in the PR. It makes sense to change locally while you're developing, but it will help us avoid an accident if this never makes it to the repo.